packages/ipfs-cli/src/commands/daemon.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
     if (argv.initConfig) {
       try {
         const raw = fs.readFileSync(argv.initConfig)
-        config = JSON.parse(raw)
+        config = JSON.parse(raw.toString())
```

Comment:
@achingbrain yes but TS can't infer that `encoding` option is passed here because there is nothing that verifies that so it rightfully assumes it could be a buffer. This seemed like a reasonable way to address this. Instead I could do a following, if you prefer

```suggestion
        config = JSON.parse(/** @type {string} */(raw))
```

Reply:
I think I may have misunderstood your comment, are you suggesting to change a call to `readFileSync` as follows:

```js
const config = JSON.parse(fs.readFileSync({
  ...argv.initConfig,
  encoding: 'utf8'
}))
```

Reply:
Yeah sorry I was silly, makes sense & I'll update.